src/main/resources/com/sap/oss/phosphor/fosstars/advice/oss/OssAdvice.yml

```diff
@@ -56,6 +56,30 @@ If a project runs Bandit scan checks for commits:
         url: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-using-a-list-of-events
       - name: An eample to trigger Bandit scan check on every pull-request.
         url: https://github.com/TNLinc/CV/blob/main/.github/workflows/bandit.yml#L3
+If a project runs GoSec scans:
+  - advice: >
+      You can open a pull request to run GoSec scans in the project using GitHub action workflow.
+    links:
+      - name: GitHub workflow action job config to run GoSec code scanning for a repository.
+        url: https://github.com/securego/gosec#github-action
+      - name: An example to run GoSec scan check as part of GitHub action workflow.
+        url: https://github.com/gofiber/fiber/blob/master/.github/workflows/security.yml#L10
```

Comment:
It is a good idea to show the example, but we focus here on the main context reference documents, not examples. As you can check this in the other advisories. I suggest you to remove this advice.